storehaus-memcache/src/main/scala/com/twitter/storehaus/memcache/MemcacheLongStore.scala

```diff
@@ -34,9 +35,16 @@ import com.twitter.bijection.netty.ChannelBufferBijection
  */
 object MemcacheLongStore {
   private implicit val cb2ary = ChannelBufferBijection
+  private implicit val b2ary = new Bijection[Buf, Array[Byte]] {
```

Comment:
From [Bijection.scala](https://github.com/twitter/bijection/blob/develop/bijection-core/src/main/scala/com/twitter/bijection/Bijection.scala#L68):
```scala
/**
 * Abstract class to ease Bijection creation from Java.
 */
abstract class AbstractBijection[A, B] extends Bijection[A, B] {
```

Doesn't this suggest we should be implementing Bijection directly?